functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configurationfile/LocalChangesNewTestsSelectionExecutionWithConfigFileFunctionalTest.java

```diff
@@ -38,7 +38,7 @@ public void should_only_execute_new_tests_related_to_single_local_change() throw
             .applyAsLocalChanges("Adds new unit test");
 
         // when
-        final TestResults actualTestResults = project.build().run();
+        final TestResults actualTestResults = project.build("config/impl-base").run();
```

Comment:
Are you sure that this will really verify the recursive lookup?
IIRC, this will still execute the build from the parent directory - which means that the maven extension will be invoked from the same directory where the configuration is placed as well. By specifying the submodules you are saying that you want to execute only this module of the whole project (setting `-pl` parameter).

Reply:
You are right. To set `-f` or working directory option from maven we need to release new version of `shrinkwrap-resolver-impl-maven-embedded` as @hemanik already fixed issue with `setWorkingDir` & added functionality for `setAlternatePomFile` in https://github.com/shrinkwrap/resolver/pull/125

Reply:
the `-f` functionality is not really necessary here. you can change the path here https://github.com/arquillian/smart-testing/blob/master/functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/ProjectBuilder.java#L63
But yeah, I can release it. I wanted to do it anyway this week...

Reply:
Looking at the code - it won't change really anything - both calls: that one I referenced and the `setAlternatePomFile` one would be placed in the same method. So you don't need to wait for the release as it doesn't bring any benefit to your code.